README.md

```diff
@@ -1535,6 +1538,96 @@ hermione_base_url=http://example.com hermione path/to/mytest.js
 hermione_browsers_firefox_sessions_per_browser=7 hermione path/to/mytest.js
 ```
 
+### Debug mode
+
+In order to understand what is going on in the test step by step, there is a debug mode. You can run tests in this mode using these options: `--inspect` and `--inspect-brk`. The difference between them is that the second one stops before executing the code.
+
+Example:
+```
+hermione path/to/mytest.js --inspect
+```
+
+**Note**: In the debugging mode, only one worker is started and all tests are performed only in it.
+Use this mode with option `sessionsPerBrowser=1` in order to debug tests one at a time.
+
+### REPL mode
+
+Hermione provides a [REPL](https://en.wikipedia.org/wiki/Read–eval–print_loop) implementation that helps you not only learn the framework API, but also debug and inspect your tests. In this mode, there is no timeout for the duration of the test (it means that there will be enough time to debug the test). It can be used when specifying the CLI options:
```

Comment:
`helps you not only learn` -> `helps you not only to learn`
`but also debug` -> `but also to debug`
`can be used when specifying` -> `can be used by specifying`